### ISSUE TITLE
Audio exploit crashing matches

### ISSUE THREAD

**Issue by FurtadoPires** [Tsuey/L4D2-Community-Update#185]
I'm probably not the first one reporting it, but better safe than sorry I think

There are some cheaters using a specific audio to crash the match every time it's played

The cheater in the video crashed some matches I was in, so I've decided to follow him and record the process
[Video of audio exploit example.zip](https://github.com/Tsuey/L4D2-Community-Update/files/8457704/Video.of.audio.exploit.example.zip) and [Virustotal repport of my zip file](https://www.virustotal.com/gui/file/e19f17ad868f483824ac1664b49ab62fbae046b9851d936b4dbac9683361f795?nocache=1)
(sorry for using .zip, github doesn't let me upload mkv directly)

This is a example of my console messages after the game crash
[Console messages](https://gist.github.com/FurtadoPires/cfb988512a58848c8a2374db87154bbd#file-gistfile1-txt)

Apparently the cheater always use the same method:
1 - Call an Alltalk vote (maybe the attack only works for those who can rear the audio, or might be part of the exploit process)
2 - Play the audio as showed in the example, two seconds latter you're back to the menu (not sure if the entire server crashes or only the players)

**Comment by shqke** [comment]
A demo would be nice to have there. 😉

**Comment by FurtadoPires** [comment]
The guy who's using it apparently has an youtube channel, if it helps...

**Comment by lunatixxx** [comment]
https://steamcommunity.com/app/550/discussions/0/3455968050308661184/#c3455968050308709255
https://forums.alliedmods.net/showpost.php?p=2590840&postcount=126

**Comment by lunatixxx** [comment]
By the way i never see it on my server and i'm blocking sv_alltalk so maybe it's related like op said.

**Comment by lunatixxx** [comment]
Could also be related to voice_loopback, i have a plugin that auto mute those who activate it (not sure why it even exist in the first place) and some players are trying to spam the command when they see that it does not work like they want.

**Comment by lunatixxx** [comment]
Yep. I'm looking the profiles muted by the plugin and they are all suspicious with vac bans or some banned for convar violation ect.
